packages/css/src/components/footer/footer.scss

```diff
@@ -1,9 +1,11 @@
+@use "../grid/grid.mixins" as grid;
```

Comment:
willen we toch maar wel meteen de namespacing goed zetten? `as grid` -> `as grid-mixins`